lib/locales/en.ts

```diff
@@ -386,8 +393,10 @@ export const locale: Locale = {
     eoghan_of_ardstraw_bishop: 'Saint Eoghan, Bishop',
     ephrem_the_syrian_deacon: 'Saint Ephrem, Deacon and Doctor of the Church',
     epiphany_of_the_lord: 'The Epiphany of the Lord',
+    epipodius_and_alexander_of_lyon_martyrs: 'Saints Epipodius and Alexander, Martyrs',
```

Comment:
Add ‘surname’ of Saint Epipodius to the key name.

```suggestion
    epipodius_of_lyon_and_alexander_of_lyon_martyrs: 'Saints Epipodius and Alexander, Martyrs',
```